neuralmonkey/config/utils.py

```diff
@@ -16,10 +16,10 @@
 
 def deprecated(func: Callable[..., T]) -> Callable[..., T]:
     def dep_func(*args, **kwargs) -> T:
-        warn("Use of deprecated function from "
-             + "'neuralmonkey.config.utils'. " +
-             "Use '{}' instead.".format(func.__module__[13:]
-                                        + '.' + func.__name__))
+        warn("Use of deprecated function from " +
```

Comment:
tady to plus vůbec být nemusí. A nejdou tyhle dva řádky náhodou napsat na jeden?